main.go

```diff
@@ -0,0 +1,45 @@
+package main
+
+import (
+	"github.com/kinvolk/mountinfo-graph/bindata"
+	"github.com/kinvolk/mountinfo-graph/migraph"
+	"html/template"
+	"log"
+	"net/http"
+)
+
+func generateFromTemplate(htmlTemplate string, w http.ResponseWriter, body string) {
```

Comment:
@krnowak yes that makes more sense, will try with non unique name first... thanks!